charts/pvc-autoresizer/Chart.yaml

```diff
@@ -18,10 +18,18 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.1
+version: 0.6.2
```

Comment:
Is this update needed?
Adding annotations is actually a change, but we normally update the version on the release time.

Reply:
Sorry, I fixed it.